tests/conftest.py

```diff
@@ -208,3 +209,14 @@ def _f(_addr, _salt, _initcode):
         return keccak(prefix + addr + salt + keccak(initcode))[12:]
 
     return _f
+
+
+def get_distinct_integer_types_combinations():
+    # TODO Replace with master list of integer types
+    types_list = ["uint8", "int128", "int256", "uint256"]
+    return [(t, rt) for (t, rt) in product(types_list, types_list) if t != rt]
+
+
+@pytest.fixture(params=get_distinct_integer_types_combinations())
+def distinct_int_types(request):
+    return request.param
```

Comment:
Hmm, yeah this isn't all that great looking, is it?

Reply:
Yea, it does make reading the tests more convoluted and having to jump through multiple files.